addons/xeh/fnc_initDisplay3DEN.sqf

```diff
@@ -6,6 +6,9 @@ private _fnc_watchDog = {
     if (!ISPROCESSED(missionNamespace)) then {
         INFO_1("missionNamespace processed [%1]", ISPROCESSED(missionNamespace));
         [] call CBA_fnc_preInit;
+
+        // Enable frame related functions (normally enabled in postInit)
```

Comment:
```suggestion
        // Enable frame-related functions (normally enabled in postInit)
```

Reply:
Thanks, I have dyslexia so this is one to keep an eye out for in my PRs :P